script.py

```diff
@@ -0,0 +1,14 @@
+import argparse
+
+from extract import repo_to_jsonl
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Extract repository.")
+
+    parser.add_argument("repo_name", type=str, help="name of extracted repository")
```

Comment:
* description should start with upper case
* `repo_name` - let's use `-` instead of `_`
* let's provide shortening - like `-r` for `--repo-name`